src/app/GlobalRedux/Features/CoinChartSlice/index.tsx

```diff
@@ -0,0 +1,70 @@
+`use client`;
+
+import { createSlice, createAsyncThunk, PayloadAction } from "@reduxjs/toolkit";
+
+interface ChartData {
+  prices: number[][];
+  market_caps: number[][];
+  total_volumes: number[][];
+}
+
+interface CoinChartState {
+  charts: {
+    [coinName: string]: {
+      [timePeriod: string]: ChartData;
+    };
+  };
+  loading: boolean;
+  error: string;
+}
+
+export const fetchCoinChart = createAsyncThunk(
+  "coinChart/getCoinChart",
+  async (
+    {
+      coinId,
+      currency,
+      timePeriod,
+    }: { coinId: string; currency: string; timePeriod: string },
+    thunkApi
+  ) => {
+    const COIN_URL = `https://api.coingecko.com/api/v3/coins/${coinId}/market_chart?vs_currency=${currency}&days=${timePeriod}`;
+    const response = await fetch(COIN_URL);
+    const data = await response.json();
+    console.log("fetch requested:", data);
```

Comment:
no

Reply:
Gone. This was from creating a branch off PR 37 instead of master.